src/components/MyKiva/JournalUpdatesCarousel.vue

```diff
@@ -19,6 +20,20 @@
 					@share-loan-clicked="shareLoanClicked"
 				/>
 			</template>
+			<template v-if="showLoadMore" #view-more>
+				<div
+					:key="`view-more-card`"
+					class="tw-flex tw-items-center tw-h-full tw-pr-3"
+				>
+					<kv-button
+						class="tw-mt-2"
+						variant="secondary"
+						@click="loadMoreUpdates"
+					>
+						Load more<br>updates
```

Comment:
Does this `br` also look right in mobile?

Reply:
Yeah, makes a smaller button there.